chapters/es/chapter2/1.mdx

```diff
@@ -0,0 +1,20 @@
+# Introduction
```

Comment:
We should also exclude all the English files from this PR. (I need to write a better guide as I realise the steps aren't super clear right now 😅 )